### DISCUSSION TITLE
将libEGL.so  libGLESv2.so libpag.so编译到项目里，其它文件需要include egl.h 就会出现编译错误

### DISCUSSION THREAD

**Discussion by tang-1210** [Tencent/libpag#2574] [Q&A] [answered]
将libEGL.so  libGLESv2.so libpag.so编译到项目里，其它文件需要include egl.h 就会pag的file.h就会报错很多未定义是为什么

**Comment by kevingpqi123** [comment, marked as answer]
libpag linux 有相关 demo，依赖是 libpag.a， 你可以参考对比下
